hockeysdk/src/main/java/net/hockeyapp/android/ExceptionHandler.java

```diff
@@ -75,6 +74,15 @@ public static void saveException(Throwable exception, Thread thread, CrashManage
             return;
         }
 
+        // Check for number of crashes on disk and don't save the crash in case we have more than 25 on disk.
```

Comment:
The limit is 50 here. Should update this comment and Line81.

Reply:
A constant could avoid further mistakes in future